feature-libs/product/configurators/rulebased/cpq/cpq.models.ts

```diff
@@ -85,7 +85,15 @@ export namespace Cpq {
    */
   export interface UpdateAttribute {
     configurationId: string;
-    stdandardAttrCode: string;
+    standardAttributeCode: string;
+    changeAttributeValue: ChangeAttributeValue;
+  }
+
+  /**
+   *
+   * An interface representing the update request body structure for update of CPQ configuration attribute.
+   */
+  export interface ChangeAttributeValue {
     attributeValueIds?: string;
```

Comment:
isn't the attributeValueId manadatory? - that should also be reflected in the model.

Reply:
attributeValueId is not mandatory. Only one of possible parameter fields has to be provided for a call of API: either attributeValueId , or userInput, or quantity

Reply:
ok - agree